docs/en/latest/plugins/aws-lambda.md

```diff
@@ -43,21 +43,16 @@ This Plugin supports authorization via AWS API key and AWS IAM secrets.
 | authorization        | object  | False    |         |              | Authorization credentials to access the cloud function.                                                                                    |
 | authorization.apikey | string  | False    |         |              | Generated API Key to authorize requests to the AWS Gateway endpoint.                                                                       |
 | authorization.iam    | object  | False    |         |              | Used for AWS IAM role based authorization performed via AWS v4 request signing. See [IAM authorization schema](#iam-authorization-schema). |
+| authorization.iam.accesskey  | string | True     |               | Generated access key ID from AWS IAM console.                                       |
+| authorization.iam.secretkey | string | True     |               | Generated access key secret from AWS IAM console.                                   |
+| authorization.iam.aws_region | string | False    | "us-east-1"   | AWS region where the request is being sent.                                         |
+| authorization.iam.service    | string | False    | "execute-api" | The service that is receiving the request. For Amazon API gateway APIs, it should be set to `execute-api`. For Lambda function, it should be set to `lambda`. |
```

Comment:
Here I've generalized `For HTTP trigger` to `Amazon API gateway APIs` because both types of APIs Amazon API gateway support (HTTP and REST) would require the `service` to be configured as `execute-api`.